wsvvrijheid/monorepo
# Donate

### How payment process works?

<img width="2194" alt="mollie" src="https://user-images.githubusercontent.com/22167684/219562889-2a0530ed-8c1b-427c-a691-22083fc4a983.png">

### How to test it locally?

> Mollie doesn't support `localhost`. You should use ngrok to have a gloal address

1. Download ngrok `brew install ngrok/ngrok/ngrok`
8. Signup ngrok https://dashboard.ngrok.com/signup
9. Set ngrok auth token https://dashboard.ngrok.com/get-started/setup `ngrok config add-authtoken <your-token>`
10. Open ngrok tunnel `ngrok http 4021`. This will give you an endpoint like `https://ce80-94-157-84-22.eu.ngrok.io` 
11. Add mollie key and ngrok urls in `.env.local`
```bash
NX_MOLLIE_KEY=test_MmPu3hpDsDq3ra4KVnhRqEtncesaWP
NX_DONATION_ENABLED=true
VERCEL_URL=<ngrok-url>
```
6. `yarn api` and `yarn foundation`

### What is missing

- [x] Partner info
- [ ] Send invoice to user
